core/test.py

```diff
@@ -1,8 +1,37 @@
+from abc import ABCMeta
+
 from unittest import TestCase
 
+from core import app
+from user.models import User
+from blog.models import Article
 from .utils import clean_html
 
 
+class BaseTest(TestCase):
+    __metaclass__ = ABCMeta
+
+    def setUp(self):
+        self.client = app.test_client()
+
+    def tearDown(self):
+        User.objects.delete()
+        Article.objects.delete()
+
+    def login_user(self):
```

Comment:
I've try to log an user with `flask_login.login_user` 
Firstly I had an issue : `RuntimeError: working outside of application context` 
So I've add `with app.test_request_context():`, even with `with app.request_context():` It has remove the `RuntimeError` but the user is still not authenticated.
I've also change the arguments of `login_user`, I'm stuck with this issue